tools/hist_utilities.py

```diff
@@ -27,7 +27,7 @@ def hist_to_value_error_tuplelist( hist ):
     return zip( values, errors )
 
 def values_and_errors_to_hist( values, errors, bins ):
-    assert( len( values ) == len( bins ) )
+    assert( len( bins ) == len( values ) + 1 )
```

Comment:
I think bins means here bin edges. In which case we should rename it. Created issue #293 for this.